core/src/stages/connect.cpp

```diff
@@ -145,7 +145,7 @@ void Connect::compute(const InterfaceState& from, const InterfaceState& to) {
 	const auto& path_constraints = props.get<moveit_msgs::msg::Constraints>("path_constraints");
 
 	const moveit::core::RobotState& final_goal_state = to.scene()->getCurrentState();
-	std::vector<robot_trajectory::RobotTrajectoryConstPtr> sub_trajectories;
+	std::vector<std::pair<std::string, robot_trajectory::RobotTrajectoryConstPtr>> trajectory_planner_vector;
```

Comment:
Rather than this being a `std::pair`, I may suggest making a struct to make things a little easier to work with?